build.cmd

```diff
@@ -52,6 +52,8 @@ set __SkipCoreLibBuild=
 set __SkipNativeBuild=
 set __SkipTestBuild=
 set __BuildSequential=
+set __SkipRestore=
+set __SkipNuget=
```

Comment:
The name SkipNuget is not clear for what the property is doing. It would be better to use `BuildPackages` (the same we have in CoreFx)